renovate.json

```diff
@@ -320,9 +320,9 @@
       "enabled": true
     },
     {
-      "groupName": "Profiling",
-      "matchDepNames": ["peggy", "@types/dagre"],
-      "reviewers": ["team:obs-ux-infra_services-team"],
```

Comment:
core / operations aren't really the right teams, we don't maintain any related parts of the UI.  Global UX is likely a closer fit.